polkadot/runtime/common/src/claims.rs

```diff
@@ -166,6 +174,13 @@ pub mod pallet {
 	#[pallet::without_storage_info]
 	pub struct Pallet<T>(_);
 
+	#[cfg(feature = "runtime-benchmarks")]
+	/// Helper trait to benchmark the `PrevalidateAttests` transaction extension.
+	pub trait BenchmarkHelperTrait<RuntimeCall, DispatchInfo> {
```

Comment:
nit but I find it a bit strange to call what is already a _`trait`_: `somethingSoemthingTrait`, if you know what I mean :))